profiles/package.mask

```diff
@@ -36,6 +36,23 @@
 
 #--- END OF EXAMPLES ---
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2025-05-13)
+# sci-electronics/gazebo depends on x11-libs/qwt:6[qt5] and Qt5.
+# Maintainer generally unresponsive on bugzilla, so lots of open bugs.
```

Comment:
I'd prefer to keep this kind of remark in the commit message but not in the package.mask, as it's exposed to users. Not that you're wrong...